test/unit/test_itde_manager.py

```diff
@@ -47,7 +48,7 @@ def env_info() -> EnvironmentInfo:
 
 
 @pytest.fixture
-def db_image_version(monkeypatch) -> str:
+def db_image_version(monkeypatch) -> Iterator[str]:
```

Comment:
Is it an iterator?

Reply:
We could also use Generator, if you prefer.